chapters/06-smart-contracts-in-ralph.md

```diff
@@ -1362,13 +1362,11 @@ Contract OldCode(owner: Address, n: U256) {
         return n
     }
 
-    @using(checkExternalCaller = false)
     pub fn migrate(newCode: ByteVec) -> () {
         checkCaller!(callerAddress!() == owner, 0)
         migrate!(newCode)
     }
 
-    // Please check owner in production code
```

Comment:
Is this comment needed since the example already uses `checkCaller!`? Wouldn't it make more sense to have it if we were not checking the caller in this example?